diff --git a/node_modules/whatwg-fetch/fetch.js b/node_modules/whatwg-fetch/fetch.js
index f39a983..b6b107c 100644
--- a/node_modules/whatwg-fetch/fetch.js
+++ b/node_modules/whatwg-fetch/fetch.js
@@ -6,6 +6,18 @@ var g =
   (typeof global !== 'undefined' && global) ||
   {}
 
+if (typeof g.DOMException === 'undefined') {
+  g.DOMException = function DOMException(message, name) {
+    this.message = message;
+    this.name = name || 'DOMException';
+    var error = Error(message);
+    this.stack = error.stack;
+  };
+  g.DOMException.prototype = Object.create(Error.prototype);
+  g.DOMException.prototype.constructor = g.DOMException;
+}
+
+
 var support = {
   searchParams: 'URLSearchParams' in g,
   iterable: 'Symbol' in g && 'iterator' in Symbol,
@@ -523,6 +535,7 @@ try {
   DOMException.prototype.constructor = DOMException
 }
 
+
 export function fetch(input, init) {
   return new Promise(function(resolve, reject) {
     var request = new Request(input, init)
